extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -325,7 +325,9 @@ export async function defaultAzureAccountServiceFactory(): Promise<vscodeMssql.I
 export async function getDefaultPublishDeploymentOptions(project: ISqlProject): Promise<mssql.DeploymentOptions | vscodeMssql.DeploymentOptions> {
 	const schemaCompareService = await getSchemaCompareService();
 	const result = await schemaCompareService.schemaCompareGetDefaultOptions();
-	const deploymentOptions = result.defaultDeploymentOptions;
+	let deploymentOptions = result.defaultDeploymentOptions;
+	deploymentOptions.optionsMapTable = new Map(Object.entries(result.defaultDeploymentOptions.optionsMapTable).map((x) => [x[0].charAt(0).toUpperCase() + x[0].slice(1), x[1]]));
```

Comment:
can x be something more descriptive, like option? Also please add a comment that explains what's happening here